build/make-csv-generator.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+if [ -x "${BASEPATH}/../operator-sdk" ]; then
+       OPERATOR_SDK="${BASEPATH}/../operator-sdk"
+else
+    which operator-sdk &> /dev/null || {
```

Comment:
for some reason this is failing, need to see if gen-operator-sdk does its work or the check here is faulty
(locally you wont see it if you have operator-sdk installed already)

Reply:
oh i see why, BASEPATH isn't defined. I'll update the PR.

Reply:
I think I fixed this now. give it a shot.